static/scss/detail/subnav.scss

```diff
@@ -34,6 +44,118 @@
       display: none;
     }
   }
+
+  .enroll-button-block {
+    position: relative;
+    @include media-breakpoint-up(md) {
+      order: 2;
+    }
+  }
+
+  .enroll-button {
+    width: 140px;
+    display: inline-flex;
+    margin: 0;
+    border-radius: 5px;
+    background-color: $primary;
+    color: white !important;
+    padding: 15px 25px;
+    text-transform: uppercase;
+    cursor: pointer;
+    font-weight: 600;
+    font-size: 16px;
+    margin-left: 20px;
+
+    &.enrolled {
+      border: 4px solid $primary;
+      background-color: white;
+      color: $primary;
+    }
+
+    &:hover {
+      // underline is at different levels since we centered the text and checkmark vertically, so it looks strange
+      text-decoration: none;
+    }
+  }
+
+  .enroll-dropdown {
+    .dropdown-toggle::after {
+      display: none;
+    }
+
+    .login-popup {
+      max-width: 350px;
+      margin-left: 20px;
+      margin-right: 20px;
+      padding: 20px;
+      font-weight: 600;
+      font-size: 20px;
+      min-width: 320px;
+
+      .close-btn {
+        position: absolute;
+        top: 0;
+        right: 0;
+        padding: 5px;
+        margin: 20px 7px 0 0;
+        font-size: 72px;
+        font-weight: 300;
+        background-color: #fff !important;
+        color: #000;
```

Comment:
be consistent in using codes and names of colors, I can see `#fff` and `white` in the same file.